dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java

```diff
@@ -130,13 +135,24 @@ private boolean userDoesNotHaveAccess(final User user) throws DotDataException {
     }
 
     @Override
-    public Map<String, Set<String>> appKeysByHost() {
-        return secretsStore.listKeys().stream()
+    public Map<String, Set<String>> appKeysByHost() throws DotSecurityException, DotDataException{
```

Comment:
I think `appKeysByHost ` methods do not need to be public. But in case it's necessary, they should have integration tests

Reply:
yeah agreed. It'll remind private